@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1441,22 +1441,27 @@ export class TextElementsRenderer {
     }
 
     private addPointLabel(
-        pointLabel: TextElement,
         labelState: TextElementState,
         groupState: TextElementGroupState,
         position: THREE.Vector3,
         screenPosition: THREE.Vector2,
         poiRenderer: PoiRenderer,
         textCanvas: TextCanvas,
         renderParams: RenderParams,
-        temp: TempParams
+        temp: TempParams,
+        iconIndex?: number
     ): boolean {
-        const textRenderState: RenderState = labelState.textRenderState!;
-        const iconRenderState: RenderState = labelState.iconRenderState!;
+        const pointLabel: TextElement = labelState.element;
+        const textRenderState: RenderState | undefined = labelState.textRenderState;
```

Comment:
can you move the textRenderState declaration to near dorendertext bool?

Reply:
Most of this code is changing in the follow-up review I mentioned, so no much use in changing this here.